packages/cli/commands/project/installDeps.js

```diff
@@ -0,0 +1,73 @@
+const {
+  installPackages,
+  getProjectPackageJsonLocations,
+} = require('../../lib/dependencyManagement');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { getProjectConfig } = require('../../lib/projects');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const path = require('path');
+const { i18n } = require('../../lib/lang');
+
+const i18nKey = `commands.project.subcommands.installDeps`;
+
+exports.command = 'install-deps [packages..]';
+// Intentionally making this null to hide command
+exports.describe = null;
+// exports.describe = uiBetaTag(i18n(`${i18nKey}.help.describe`), false);
+exports.builder = yargs => {
+  yargs.example([
+    ['$0 project install-deps', i18n(`${i18nKey}.help.installAppDepsExample`)],
+    [
+      '$0 project install-deps dependency1 dependency2',
+      i18n(`${i18nKey}.help.addDepToSubComponentExample`),
+    ],
+  ]);
+};
+
+exports.handler = async ({ packages }) => {
+  try {
+    const projectConfig = await getProjectConfig();
```

Comment:
Can you add a call to `trackCommandUsage` somewhere at the top of this?

Reply:
Good catch, are there any naming conventions for those? `project-install-deps` ok?

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1133/commits/2a9e7834c1260fd0cd49ac2908f6547f61804361